app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -172,6 +176,16 @@ public void run() {
         }
     };
 
+    private void deleteAlarm(int id) {
+
+        Intent intent = new Intent(this,Notification.class);
+        PendingIntent pendingIntent = PendingIntent.getBroadcast(this,id,intent,PendingIntent.FLAG_UPDATE_CURRENT);
+        AlarmManager alarmManager = (AlarmManager) getSystemService(ALARM_SERVICE);
+        alarmManager.cancel(pendingIntent);
+        Log.i(TAG, "deleteAlarm: Alarm deleted for id "+id);
+
+    }
```

Comment:
There is already a `deleteAlarm` function in `LoyaltyCardEditActivity`. If you are going to reuse the same code in multiple classes, please put it in Utils if there is no better other place.